manifests/init.pp

```diff
@@ -17,6 +19,20 @@
     purge   => $config_dir_purge,
   }
 
+  if str2bool($cron_purge) {
```

Comment:
Sorry, I've been on vacation. I think I attempted to just do an "if $cron_purge == True {" but it didn't play out properly. It could be because the true/false is getting passed to the function as a string instead of a Boolean value. 

Sent from my computerphone from the future.

> On Feb 10, 2015, at 11:08 AM, Joshua Hoblitt <notifications@github.com> wrote:
> 
> In manifests/init.pp:
> 
> > @@ -17,6 +19,20 @@
> >      purge   => $config_dir_purge,
> >    }
> >  
> > +  if str2bool($cron_purge) {
> I'm not following... validate_bool() is in place as a guard function so str2bool() could only be operating on a bool.
> 
> —
> Reply to this email directly or view it on GitHub.
>